pages/enroll/enroll.ejs

```diff
@@ -15,53 +21,93 @@
-                                   <input type=&quot;hidden&quot; name=&quot;course_instance_id&quot;
-                                          value=&quot;<%= course_instance.course_instance_id %>&quot;>
-                                   <button type=&quot;submit&quot; class=&quot;btn btn-danger&quot;>
-                                     Remove <%= course_instance.short_label %>
-                                   </button>
-                                 </form>
-                                 ">
+                   data-toggle="modal" data-target="#removeModal<%= modalIdx %>" >
                   Remove course
                 </a>
+                <div class="modal fade" id="removeModal<%= modalIdx %>" tabindex="-1" role="dialog" aria-labelledby="removeModal<%= modalIdx %>Title" aria-hidden="true">
+                  <div class="modal-dialog modal-dialog-centered" role="document">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <h5 class="modal-title" id="removeModal<%= modalIdx %>Title">Confirm remove course</h5>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                      </div>
+                      <div class="modal-body">
+                        <p>Are you sure you want to remove this course content from your PrairieLearn account?</p>
+                        <p class="modal-course-title-style"><%= course_instance.label %></p>
+                        <p>Adding or removing courses here only affects what is visible to you on PrairieLearn. This does not change your university course registration.</p>
+                      </div>
+                      <div class="modal-footer">
+                         <form name="unenroll-form" method="POST">
+                           <input type="hidden" name="__action" value="unenroll">
+                           <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                           <input type="hidden" name="course_instance_id"
+                                  value="<%= course_instance.course_instance_id %>">
+                           <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                           Cancel
+                           </button>
+                           <button type="submit" class="btn btn-danger">
+                             Remove <%= course_instance.short_label %>
+                           </button>
+                         </form>
+                      </div>
+                    </div>
+                  </div>
+                </div>
                 <% } %>
               </td>
             </tr>
-            <% }); %>
+            <%# incrementing modalIdx must be done before closing the forEach function block: %>
+            <% ; modalIdx++; }); %>
```

Comment:
Why do we have a leading semi-colon here?

Reply:
I think that's left over from an earlier draft of the code where it seemed to make the EJS easier to read/more defensive against someone putting an IIFE in front of it by accident. We can remove it.

Reply:
makes sense, removing is probably a good idea